scripts/validate_docstrings.py

```diff
@@ -471,6 +471,45 @@ def parameter_desc(self, param):
                 desc = desc[:desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
```

Comment:
Why does this method even need to exist at all? Couldn't you just use a regular expression to remove directives?

Reply:
I wanted to create a method that returns the parameter description as list of lines, because the method `parameter_desc` only return the entire description as one unique string.

This is because `has_proper_punctuation` needs to check if the last line of the description is part of a bullet point. (The last character of the last line can also not be a period if the last line is part of a bullet point.)